### DISCUSSION TITLE
Capacity of an RNW App ? have serveral questions about RNW to figure out what i able to do with it ?

### DISCUSSION THREAD

**Discussion by NguyenHoangMinhkkkk** [microsoft/react-native-windows#13274] [Q&A] [answered]
Hi, i'm currently a Mobile developer working with RN for mobile and
I have serveral questions about RNW to figure out what i able to do with it ?

1/ Can i deploy a released App without microsoft store ? i mean i can grab the release build (like  .apk | .dmg | .ipa) to install App on other window computer with setup env & nodejs...etc on target computer ?

2/ RNW deployment release build able to run on window Server computer? Because my deployment targets mostly running Window Server 2012, 2016, 2022... and they running without internet.  _3/ For If Window Server availabe... which version of Window Server OS  are compatiple ?_

4/ Can RNW app access and make change with Fs like: Move files, copy files, read files, create folder, delete files ...
5/ Can RNW run as startup app, whether the computer restart or shutdown, it can live on and do its work continuous?

Thanks for reading and help me answer my questions

**Comment by jonthysell** [comment, marked as answer]
1. Yes
2. Yes, some versions.
3. See below about supported windows versions
4. Through native modules like other RN apps, yes.
5. Maybe?

Maybe these docs can help you:

- https://microsoft.github.io/react-native-windows/docs/win10-compat
- https://microsoft.github.io/react-native-windows/docs/native-code
- https://en.wikipedia.org/wiki/List_of_Microsoft_Windows_versions#Server_versions

**Reply by NguyenHoangMinhkkkk** [reply]
Thank you so much for answering my questions.
